api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TxMlQueries.java

```diff
@@ -0,0 +1,64 @@
+package org.openmrs.module.eptsreports.reporting.library.queries;
+
+public class TxMlQueries {
+
+  public static String getPatientsWhoMissedAppointment(
+      int min,
+      int max,
+      int returnVisitDateForDrugsConcept,
+      int returnVisitDate,
+      int pharmacyEncounterType,
+      int adultoSequimento,
+      int arvPediatriaSeguimento) {
+
+    return "SELECT patient_id FROM (SELECT p.patient_id,MAX(encounter_datetime) encounter_datetime FROM patient p "
```

Comment:
@ningosi i recommend we use `String.format()` here to make this queries more readable